templates/README

```diff
@@ -1 +1,60 @@
-This contains some basic description about the BIDS dataset.
+### Introduction
+The README is usually the starting point for researchers using your data
+and serves as a guidepost for users of your data. A clear and informative
+README makes your data much more usable.
+
+Overview
+
+A paragraph giving an overview of the experiment. This should include the 
+goals or purpose and a discussion about how the experiment tries to achieve
+these goals.
+
+A brief overview of the tasks in the experiment.
+
+A brief discussion of condition variables (sometimes called contrasts
+or independent variables) that were varied across the experiment.
+
+A brief discussion of the response variables (sometimes called the
+dependent variables) that were measured and or calculated to assess
+the effects of varying the condition variables. This might also include
+questionnaires administered to assess behavioral aspects of the experiment.
+
+A brief discussion of the control variables --- that is what aspects 
+were explicitly controlled in this experiment. The control variables might 
+include subject pool, environmental conditions, set up, or other things 
+that were explicitly controlled.
+
+More details on all of this should be included in the Methods section.
+
+
+Methods  
+
+Subjects: A brief sentence about the subject pool in this experiment.
+ 
+Apparatus: A summary of the equipment and environment setup for the
+experiment. For example, was the experiment performed in a shielded room
+with the subject seated in a fixed position.
+
+Initial setup: A summary of what setup was performed when a subject arrived.
+
+Task organization: How the tasks were organized for a session.
+This is particularly important because BIDS datasets usually have task data
+separated into different files.) What was the order that tasks were performed? 
+Was task order counter-balanced?  What other activities were interspersed 
+because the tasks?  
+
+Task details: As much detail as possible about the task and the
+events that were recorded. 
+
+Additional data acquired: A brief indication of data other than the
+imaging data that was acquired as part of this experiment. In addition
+to data from other modalities and behavioral data, this might include
+questionnaires and surveys, swabs, and clinical information. Indicate
+the availability of this data.
+
+Experimental Location: This should include the geographical 
+location and facility.
+
+Notes: Any additional information or pointers to information that
+might be helpful to users of the dataset.
+
```

Comment:
Maybe mention something about missing data for some subject or any noticeable feature of the data for certain participants. Saying this as the data for a participant in one of the dataset I shared with some colleagues was missing half of the visual cortex: something I knew but I had to let them discover for themselves because I had not put it into the README.

Reply:
☝️ this! I think a place to add extra information about the acquisitions that does not fit e.g., the `participants.tsv` or `_sessions.tsv` files (usually because it's more "qualitative" data) would be very good to have in BIDS!

but not sure if the `README` is a good place for that. from my experience, quite a few of data acquisitions don't run as smoothly as we hope, so depending on how meticulously research staff takes notes, the README would become quite long.

for reference, I have posted a similar question [on neurostars before](https://neurostars.org/t/where-in-a-bids-dataset-should-i-put-notes-about-individual-mri-acqusitions/17315).

Reply:
Cool!! Thanks for that @lnnrtwttkhn

I think we can at least create a section in the README about this. And as the issue in neurostars, the template can have some of the explanations from the neurostars thread.

Reply:
+1 - this list can also be built out and iterated on over time as more helpful suggestions are surfaced